utils/llm.py

```diff
@@ -196,13 +193,13 @@ def log_llm_call(run_name: str, agent_name: str, debug_level: int, label: str =
     save_dir = _llm_debug_dir.joinpath(run_name, agent_name)
     save_dir.mkdir(parents=True, exist_ok=True)
 
-    # Sanitize timestamp
-    sanitized_timestamp = datetime.now().strftime("%Y-%m-%d_%H_%M_%S_%f")
-
     # Write content of LLM call to file
     context = _llm_debug_params.pop("messages")
     response_text = _llm_debug_params.pop("response")
-    save_path = save_dir.joinpath(f"{sanitized_timestamp}{'-' + label if label is not None else ''}.txt")
+    save_name = datetime.now().strftime("%Y-%m-%d_%H_%M_%S_%f")
```

Comment:
Minor thing, but maybe the format `"%Y-%m-%d_%H-%M-%S-%f"` would be more legible? The switch to underscores starting with the separator between the date and time makes it a little hard to parse at times.